/teleop:
  ros__parameters:
    frequency: 100 
    mode: "cmd_vel_world" # one of cmd_rpy, cmd_vel_world
    cmd_rpy:
      x_velocity_axis: 5
      y_velocity_axis: 4
      z_velocity_axis: 2
      yaw_velocity_axis: 1
      x_velocity_max: 30.0 # deg
      y_velocity_max: 30.0 # deg
      z_velocity_max: 60000.0 # PWM
      yaw_velocity_max: 200.0  # deg/s
    cmd_vel_world:
      x_velocity_axis: 5
      y_velocity_axis: 4
      z_velocity_axis: 2
      yaw_velocity_axis: 1
      x_velocity_max: 0.5 # m/s
      y_velocity_max: 0.5 # m/s
      z_velocity_max: 0.5 # m/s
      yaw_velocity_max: 0.5  # rad/s
      x_limit: [-1.0, 1.0] # m
      y_limit: [-1.0, 1.0] # m
      z_limit: [0.0, 2.0] # m
    initial_position:
      x: 0.0 # m
      y: -0.5 # m
      z: 0.10 # m





